03_capabilities.html

```diff
@@ -1 +1,179 @@
-<!-- Section removed as it was describing specific local end implementations -->
\ No newline at end of file
+<section>
+  <h2>Capabilities</h2>
+  <p>
+    Webdriver <dfn>capabilities</dfn> allow the <a>local end</a> to specify what
+     features it requires the <a>remote end</a> to fulfil to be able to create a
+      <a>new session</a>
+  </p>
+  <h3>Processing Capabilities</h3>
+  <p>
+    <p>The <a>remote end steps</a> to <dfn>process capabilities</dfn> are:</p>
+    <ol>
```

Comment:
I'd call these just “capabilities” and call the parameter passed in “desired capabilities” or something similar.